seqr/views/apis/staff_api_tests.py

```diff
@@ -260,6 +271,105 @@
+                    "Mendelian Rare Disease Exome"
+                ],
+                "dbgap_submission": [
+                    "WES",
+                    "Array"
+                ]
+            },
+            "createdTime": "2019-09-09T19:21:12.000Z"
+        },
+        {
+            "id": "rec2Nkg10N1KssPc3",
+            "fields": {
+                "SeqrCollaboratorSampleID": "19F-DR-2",
+                "CollaboratorSampleID": "VCGS_FAM203_621_D2",
+                "Collaborator": ["reca4hcBnbA2cnZf9"],
+                "dbgap_study_id": "dbgap_stady_id_2",
+                "dbgap_subject_id": "dbgap_subject_id_2",
+                "dbgap_sample_id": "SM-JDBTT",
+                "SequencingProduct": [
+                    "Standard Germline Exome v6 Plus GSA Array"
+                ],
+                "dbgap_submission": [
+                    "WES",
+                    "Array"
+                ]
+            },
+            "createdTime": "2019-07-16T18:23:21.000Z"
+        }
+    ]}
+
+AIRTABLE_COLLABORATOR_RECORDS = {
+    "records": [
+        {
+            "id": "recW24C2CJW5lT64K",
+            "fields": {
+                "CollaboratorID": "Hildebrandt",
+            }
+        },
+        {
+            "id": "reca4hcBnbA2cnZf9",
+            "fields": {
+                "CollaboratorID": "Seidman",
+            }
+        }
+    ]
+}
+
+EXPECTED_PI_SUBJECT_FILE = [
+    u'1kg project n\xe5me with uni\xe7\xf8de_PI_Subject',
+    ['entity:subject_id', 'subject_id', 'prior_testing', 'project_id', 'pmid_id',
+     'dbgap_submission', 'dbgap_study_id', 'dbgap_subject_id', 'multiple_datasets', 'sex',
+     'ancestry', 'ancestry_detail', 'age_at_last_observation', 'phenotype_group', 'disease_id',
+     'disease_description', 'affected_status', 'onset_category', 'age_of_onset', 'hpo_present',
+     'hpo_absent', 'phenotype_description', 'solve_state']
+]
+EXPECTED_PI_SUBJECT_ROW = {
+    'project_guid': u'R0001_1kg', 'num_saved_variants': 0, 'dbgap_submission': 'No',
+    'hpo_absent': u'HP:0011675|HP:0001674|HP:0001508', 'solve_state': 'Unsolved', 'phenotype_group': '', 'sex': 'Male',
+    'phenotype_description': '', 'ancestry': '', 'ancestry_detail': '', 'entity:subject_id': u'NA19675_1',
+    'hpo_present': u'HP:0001631|HP:0002011|HP:0001636', 'multiple_datasets': 'No', 'onset_category': u'Adult onset',
+    'subject_id': u'NA19675_1', 'family_guid': u'F000001_1', 'affected_status': 'Affected', 'pmid_id': '',
+    'project_id': u'1kg project n\xe5me with uni\xe7\xf8de'}
+
+EXPECTED_PI_SAMPLE_FILE = [
+    u'1kg project n\xe5me with uni\xe7\xf8de_PI_Sample',
+    ['entity:sample_id', 'subject_id', 'sample_id', 'dbgap_sample_id', 'sample_source',
+     'sample_provider', 'data_type', 'date_data_generation']
+]
+EXPECTED_PI_SAMPLE_ROW = {'entity:sample_id': u'NA19675_1', 'data_type': u'WES', 'subject_id': u'NA19675_1', 'sample_provider': '', 'sample_id': u'NA19675', 'date_data_generation': '2017-02-05'}
```

Comment:
You need to test against a row that has the airtable data, so pick a row to use that has sample_provider set

Reply:
I updated the mock responses from Airtable by matching the sample ID. It seems work now. Please review.